pkg/oc/bootstrap/docker/up.go

```diff
@@ -1011,17 +1016,14 @@ func (c *ClientStartConfig) InstallMetrics(out io.Writer) error {
 // CheckServiceCatalogPrereqVersion ensures the OpenShift server version is high enough to
 // run the service catalog.
 func (c *ClientStartConfig) CheckServiceCatalogPrereqVersion(out io.Writer) error {
-	serverVersion, _ := c.OpenShiftHelper().ServerPrereleaseVersion()
-	// 3.6.0-alpha2 was the last release that did not allow the creation of namespace rolebindings w/o first
-	// creating a policybinding object.  This limitation prevents the service catalog template from instantiating.
-
-	// special case for someone who is building a local image using commits based on 3.6.0.alpha2.  They most likely
-	// have the necessary changes.
-	if serverVersion.EQ(openshiftVersion36alpha2) && (serverVersion.String() != openshiftVersion36alpha2.String()) {
+	if c.ImageVersion == "latest" {
```

Comment:
> we don't know what "openshift/origin:latest" is on their local machine so i think you still need to check the server version. passing "--version=latest" to oc cluster up does not enforce a pullalways behavior.

That would block this pull on a new tag to be able to better handle cases where someone built the latest `oc` from source, did not build the images, did not pull new images, and ran with `--service-catalog=true`.  Would you rather block on on that edge or merge this (which will work for most sane cases against head) and take a TODO?